launcher/ui/pages/instance/OtherLogsPage.cpp

```diff
@@ -374,6 +396,28 @@ void OtherLogsPage::setControlsEnabled(const bool enabled)
     ui->btnClean->setEnabled(enabled);
 }
 
+QStringList OtherLogsPage::getPaths()
+{
+    QDir baseDir(m_basePath);
+
+    QStringList result;
+
+    for (QString searchPath : m_logSearchPaths) {
+        QDirIterator iterator(searchPath, QDir::Files | QDir::Readable);
+
+        while (iterator.hasNext()) {
+            const QString name = iterator.next();
+
+            if (!name.endsWith(".log") && !name.endsWith(".log.gz"))
```

Comment:
txt is also a thing

Reply:
it will show options.txt

Reply:
Agh crash-reports I guess

Reply:
Added it - it just does not apply for the game root